numpy/f2py/tests/test_module_doc.py

```diff
@@ -0,0 +1,22 @@
+import os
+import sys
+import pytest
+
+from . import util
+from numpy.testing import assert_equal, IS_PYPY
+
+
+def _path(*a):
+    return os.path.join(*((os.path.dirname(__file__),) + a))
+
+
+class TestModuleDocString(util.F2PyTest):
+    sources = [_path('src', 'module_data', 'module_data_docstring.f90')]
+
+    @pytest.mark.skipif(sys.platform=='win32',
+                        reason='Fails with MinGW64 Gfortran (Issue #9673)')
+    @pytest.mark.xfail(IS_PYPY,
+                       reason="PyPy cannot modify tp_doc after PyType_Ready")
+    def test_module_docstring(self):
+        expected = "i : 'i'-scalar\nx : 'i'-array(4)\na : 'f'-array(2,3)\nb : 'f'-array(-1,-1), not allocated\x00\nfoo()\n\nWrapper for ``foo``.\n\n"
```

Comment:
This would be a good use of `textwrap.dedent('"""...""")`

Reply:
Thanks for the suggestion, I was not familiar with that :)